src/platform_impl/linux/wayland/event_loop/proxy.rs

```diff
@@ -2,7 +2,7 @@
 
 use std::sync::mpsc::SendError;
 
-use sctk::reexports::calloop::channel::Sender;
+use calloop::channel::Sender;
```

Comment:
That's tough, I'm not sure how we should handle such sharing. In general, I'd probably make sharing of `calloop` between drm and wayland backend, so they stay on the same version.

Reply:
Fixed